.circleci/config.yml

```diff
@@ -1,38 +1,81 @@
-# Javascript Node CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-javascript/ for more details
-#
-#
-version: 2
-jobs:
-  build:
-    docker:
-      # specify the version you desire here
-      - image: circleci/node:10.16
-
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
-      # - image: circleci/mongo:3.4.4
+version: 2.1
 
-    working_directory: ~/repo
+workflows:
+  build-test:
+    jobs:
+      - prep-deps
+      - test-build:
+          requires:
+            - prep-deps
+      - test-lint:
+          requires:
+            - prep-deps
+      - test-unit:
+          requires:
+            - prep-deps
+      - all-tests-pass:
+          requires:
+            - test-build
+            - test-lint
+            - test-unit
```

Comment:
I take it integration tests are listed under `unit` here?

Reply:
We can add a new job for integration tests, I don't believe there are any now